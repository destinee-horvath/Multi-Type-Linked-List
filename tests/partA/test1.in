NEW 5
element1
3
4.5
carrots
m
NEW 4
penguins
c

6
VIEW ALL
TYPE 0
VIEW 0
TYPE 1
VIEW 1