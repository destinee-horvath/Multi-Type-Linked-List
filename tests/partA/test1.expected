List 0: element1 -> 3 -> 4.50 -> carrots -> m
List 1: penguins -> c ->   -> 6
Number of lists: 2
List 0
List 1
string -> int -> float -> string -> char
element1 -> 3 -> 4.50 -> carrots -> m
string -> char -> char -> int
penguins -> c ->   -> 6
